depends/packages.md

```diff
@@ -162,6 +162,9 @@ From the [Gentoo Wiki entry](https://wiki.gentoo.org/wiki/Project:Quality_Assura
 >  creates. This leads to massive overlinking, which is toxic to the Gentoo
 >  ecosystem, as it leads to a massive number of unnecessary rebuilds.
 
+Where possible, packages are built with Position Independant Code. Either using
+the autotools `--with-pic` flag, or `DCMAKE_POSITION_INDEPENDENT_CODE` with CMake.
```

Comment:
I believe that `-DCMAKE_POSITION_INDEPENDENT_CODE=ON` should be added somewhere in `funcs.mk`:https://github.com/bitcoin/bitcoin/blob/e037c4fe0914d8fa9149ce7532c0d70f738e79e9/depends/funcs.mk#L173-L187

Otherwise, the PIC is not applied consistently across all static libraries in depends (capnp, multiprocess and their native counterparts for now).

---

nit: maybe
```suggestion
the autotools `--with-pic` flag, or `-DCMAKE_POSITION_INDEPENDENT_CODE=ON` with CMake.
```
?

Reply:
> I believe that -DCMAKE_POSITION_INDEPENDENT_CODE=ON should be added somewhere in funcs.mk:

This PR is for Autotools, not CMake. The CMake PR is #29706.

Reply:
As for the nit, I don't think we need to show exactly how to use the option, naming it should be fine.

Reply:
To just name an option, the `D` flag should be skipped. If "built with Position Independant Code" was mentioned as a goal, the CMake equivalent of Autotools' `--with-pic` is exactly the suggested variant. The current variant does not follow either goal.

However, it's just a minor nit, and I'm going to ACK this PR as it is after finishing my Guix build.

Reply:
I can drop the D in whichever of the conflicting PRs is merged second.

Reply:
Addressed in #29706.